tests/Application/config/packages/webpackl_encore.yml

```diff
@@ -0,0 +1,5 @@
+webpack_encore:
+    output_path: '%kernel.project_dir%/public/build/default'
+    builds:
+        shop: '%kernel.project_dir%/public/build/shop'
```

Comment:
why in assets.yaml we have 4 positions, but here we have only two? As I remember, the counts usually are equal

Reply:
Fixed